tests/platform_tests/args/advanced_reboot_args.py

```diff
@@ -120,3 +120,10 @@ def add_advanced_reboot_args(parser):
         default="",
         help="Specify the target image to restore to, or stay in target image if empty",
     )
+
+    parser.addoption("--sad_case_list",
+        default="sad, multi_sad, sad_bgp, sad_lag_member, sad_lag, sad_vlan_port, sad_inboot",
```

Comment:
Could this be changed to `", ".join(SAD_CASE_LIST)`, so that if the list changes, it doesn't have to be changed in two places, and this will always by default run all test cases?

Reply:
Good suggestion. Updated this in the latest commit. Thanks!